data-canary/scripts/talkactions/player/server_info.lua

```diff
@@ -1,11 +1,32 @@
 local serverInfo = TalkAction("!serverinfo")
 
 function serverInfo.onSay(player, words, param)
-	player:sendTextMessage(MESSAGE_EVENT_ADVANCE, "Server Info:"
-	.. "\nExp rate: " .. getRateFromTable(experienceStages, player:getLevel(), configManager.getNumber(configKeys.RATE_EXPERIENCE))
-	.. "\nSkill rate: " .. configManager.getNumber(configKeys.RATE_SKILL)
-	.. "\nMagic rate: " .. configManager.getNumber(configKeys.RATE_MAGIC)
-	.. "\nLoot rate: " .. configManager.getNumber(configKeys.RATE_LOOT))
+	local exprate = configManager.getNumber(configKeys.RATE_EXPERIENCE)
```

Comment:
why are creating a variable, if you don't use it?

Reply:
I don't understand why you're saying if I'm not going to use this code, I created it to improve optimization and internal organization. Every pull request I submit is for my own use and to function correctly and contribute to the improvement of the code.